amethyst_window/src/config.rs

```diff
@@ -124,41 +120,29 @@ impl DisplayConfig {
     /// The `MonitorsAccess` is needed to configure a fullscreen window.
     pub fn into_window_builder(self, monitors: &impl MonitorsAccess) -> WindowBuilder {
         let attrs = WindowAttributes {
-            dimensions: self.dimensions.map(Into::into),
-            max_dimensions: self.max_dimensions.map(Into::into),
-            min_dimensions: self.min_dimensions.map(Into::into),
+            inner_size: self.dimensions.map(Into::into),
+            max_inner_size: self.max_dimensions.map(Into::into),
+            min_inner_size: self.min_dimensions.map(Into::into),
             title: self.title,
             maximized: self.maximized,
             visible: self.visibility,
             transparent: self.transparent,
             decorations: self.decorations,
             always_on_top: self.always_on_top,
             window_icon: None,
-            fullscreen: self.fullscreen.map(|ident| ident.monitor_id(monitors)),
+            fullscreen: self
+                .fullscreen
+                .map(|ident| Fullscreen::Borderless(ident.monitor_handle(monitors))),
             resizable: self.resizable,
-            multitouch: self.multitouch,
         };
 
         let mut builder = WindowBuilder::new();
         builder.window = attrs;
 
         if self.loaded_icon.is_some() {
             builder = builder.with_window_icon(self.loaded_icon);
-        } else if let Some(icon) = self.icon {
-            let icon = match Icon::from_path(&icon) {
-                Ok(x) => Some(x),
-                Err(e) => {
-                    error!(
-                        "Failed to load window icon from `{}`: {}",
-                        icon.display(),
-                        e
-                    );
-
-                    None
-                }
-            };
-
-            builder = builder.with_window_icon(icon);
+        } else if let Some(_) = self.icon {
+            warn!("DisplayConfig specified Icon, but Icon loading is currently not implemented.");
```

Comment:
Is the code from the [icon loading example](https://github.com/rust-windowing/winit/blob/31110be3960bb97caeacdd0a8cbad292722f94f1/examples/window_icon.rs#L43) sufficient to retain this feature?